test/src/edit/DistortableImageEditSpec.js

```diff
@@ -5,12 +5,12 @@ describe("L.DistortableImage.Edit", function() {
 	beforeEach(function(done) {
 		map = L.map(L.DomUtil.create('div', '', document.body)).setView([41.7896,-87.5996], 15);
 
-		overlay = new L.DistortableImageOverlay('/examples/example.png', {
+		overlay = L.distortableImageOverlay('/examples/example.png', {
 			corners: [
-				new L.LatLng(41.7934, -87.6052),
```

Comment:
Thank you for the PR!! This is great.

Just one thing: can you please put back all of the `new` keywords before `L.LatLng` lines?

Really sorry!!! The issue name and description refers directly to the  L.DistortableImageOverlay class, so it was my intention to only update that -- I oversaw that the additional `new`s in the code blocks I  provided would obviously lead someone to delete them there too 🙈

I would also be happy to correct this myself if it is a hassle for you since it's my fault. 

Just make a new commit with the changes or comment back here if you'd prefer I fixed it. It's your FTO so leaving it up to you!

Reply:
Hi, I think it's mainly my fault that I didn't catch these silly mistakes. 

 overlay = L.distortableImageOverlay('/examples/example.png', { 
 	corners: [ 
 		new L.LatLng(41.7934, -87.6052), 
 		new L.LatLng(41.7934, -87.5852), 
 		new L.LatLng(41.7834, -87.5852), 
 		new L.LatLng(41.7834, -87.6052) 
 	] 
 }).addTo(map); 

Is this the correction you are looking for? Again, this is really some basic programming concept that I can't take consideration into. Another question, do I need to complete those 5 tasks from the automatic comments? For example, attach GIFS or test with grunt?

Reply:
Yes! 

After the first question I am not understanding the rest. Please clarify if you have not already answered these yourself as I see you fixed the issue correctly already. 

I'm merging this but I will not resolve the conversation so you can still reply back here 👍